addons/mail/static/tests/discuss_app/sidebar.test.js

```diff
@@ -857,10 +857,10 @@ test("chat - states: open manually by clicking the title", async () => {
     await start();
     await openDiscuss();
     await contains(".o-mail-DiscussSidebarCategory-chat .oi-chevron-right");
-    await contains(".o-mail-DiscussSidebar button", { count: 0, text: "Mitchell Admin" });
+    await contains(".o-mail-DiscussSidebarChannel", { count: 0, text: "Mitchell Admin" });
```

Comment:
I guess we no longer have the time (or maybe we do?) but this rename could have been done as a prep PR.